### ISSUE TITLE
try linear warm-up and higher peak learning rate

### ISSUE THREAD

**Issue by matsen** [matsengrp/netam#39]
> For stabilizing and enhancing training, we used a linear warm-up for 1k steps, a peak learning rate of 0.0004, a cosine
learning rate decay over 9k steps, and a weight decay of 0.01.
